### PR TITLE
feat: support parse interpolation in `WXAttribute`

### PR DESCRIPTION
**Pull request by dhlolo**
As title said.

- [ ] Documentation
- [x] Unit Tests
- [x] Code complete

And there are still a few points should be known and maybe  discuss:
  - 2 kinds of new AST node -- "WXAttributeValInterpolation" and "WXAttributeValue "
  - though it support interpolation in attribute value, deeper nodes paths also happened
  - I move property "quote" from WXAttribute node to WXAttributeValue node

### CODE REVIEW

**Review by iChenLei — COMMENTED** [review]
First of all, appreciate for your awesome work ! I have a little request, can you add new unit test for the new feature ? I noticed that some unit tests updated but no new added unit test.

**Comment by iChenLei** [comment]
`wxml-parser` is `eslint-plugin-wxml`'s direct dependency, we also need test `eslint-plugin-wxml` (but you don't need care this, i will do it).

File: src/cst/lexer.ts [hunk 1]
```diff
@@ -83,16 +87,60 @@ const INVALID_OPEN_INSIDE = createToken({
...
+const WXS_SLASH_CLOSE = createToken({
+  name: "WXS_SLASH_CLOSE",
+  pattern: /<\/(( |\t|\n|\r\n)*)wxs(( |\t|\n|\r\n)*)>/,
+  pop_mode: true,
+});
+// let callTimes = 0;
 
-const INTPN = createToken({ name: "INTPN", pattern: /((?!('|"|}})).)+/ });
+// FIXME: chevrotain回调似乎有内存泄漏，随着调用次数增多，开销明显增大，不适合大型项目
+// function matchWXS(text, startOffset) {
```
Comment (dhlolo) [inline src/cst/lexer.ts:115]: It turns to be unbearably inefficient when I use callback for pattern.

File: src/cst/lexer.ts [hunk 2]
```diff
@@ -157,10 +271,31 @@ const wxmlLexerDefinition = {
...
       SLASH,
       EQUALS,
-      STRING,
+      PURE_STRING,
+      DOUBLE_QUOTE_START,
+      SINGLE_QUOTE_START,
+      NAME,
+      SPACE,
+    ],
+    WXS_INSIDE: [
```
Comment (dhlolo) [inline src/cst/lexer.ts:280]: Pattern used to distinguish <wxs> tags like <script> tags

**Review by iChenLei — APPROVED** [review]

**Pull request merged**